.gitmodules

```diff
@@ -6,7 +6,7 @@
 	url = https://github.com/google/benchmark.git
 [submodule "third_party/cpp-btree"]
 	path = third_party/cpp-btree
-	url = https://github.com/algorithm-ninja/cpp-btree
+	url = https://github.com/mrks/cpp-btree
```

Comment:
The original repo is outdated. My fork is based on https://github.com/dmeister/cpp-btree and fixes the use of allocator::rebind, which was deprecated in C++17 and removed in C++20.